### ISSUE TITLE
Wayland support

### ISSUE THREAD

**Issue by Ireozar** [buzz/twitch-indicator#26]
When trying to run this program I get the following error (it gets output twice):
```
twitch-indicator:3460794): Gtk-CRITICAL **: 13:51:28.668: gtk_widget_get_scale_factor: assertion 'GTK_IS_WIDGET (widget)' failed
```
I am running the AUR package on Arch Linux (duh) with Wayland ([RiverWM](https://codeberg.org/river/river)).

Additional info (might be useful):
- I use [Waybar](https://github.com/Alexays/Waybar)

**Comment by buzz** [comment]
Yep, this [has been reported already](https://github.com/buzz/twitch-indicator/issues/22#issuecomment-2184891706).

I want to make the switch to Wayland soon. This is one of the blockers, so it needs to get fixed.

**Comment by 5ucur** [comment]
It does "work" for me on KDE with Wayland, in a way. It auto-launches on boot as has been set up, and I get the notifications properly, but the panel icon is missing.
